<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="coffee_table_data">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>661</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_coffee_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_coffee_btn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
